src/components/DpDataTable/DpTableHeader.vue

```diff
@@ -50,14 +50,18 @@
       scope="col"
       class="c-data-table__cell--narrow"
       @click="toggleExpandAll()">
-      <dp-wrap-trigger :title="translations.headerExpandHint" />
+      <dp-wrap-trigger
+          data-cy="isExpandableWrapTrigger"
+          :title="translations.headerExpandHint" />
```

Comment:
indentation please

Reply:
Sorry
Done in https://github.com/demos-europe/demosplan-ui/pull/812/commits/5a25e0827b2af71c1085201891d8c1a53e6c9c1b